leftwm-core/src/command.rs

```diff
@@ -32,6 +32,7 @@ pub enum Command {
         tag: TagId,
         swap: bool,
     },
+    GoToNextEmptyTag,
```

Comment:
Might want to rename this "FocusNextEmptyTag" to keep consistent with "FocusNextTag". Not sure if we'd also want to add a FocusPreviousEmptyTag as well, but something to consider.

One other idea would be to use FocusNextTag and FocusPreviousTag, but make them have an optional boolean:
```rs
FocusNextTag(Option<bool>),
FocusPreviousTag(Option<bool>)
```

Reply:
Name change seems like the best option to me, ill work on doing that when im available. My thing with making it an optional parameter is that most of the other optional parameters for commands are for changing the behavior of the command (e.g. the swap parameter for GotoTag), while going to the next empty tag feels different enough from the FocusNextTab command that an average user probably won't intuit that this functionality is contained in it, potentially making it more difficult for a user glancing at the documentation to learn that this feature is even an option. also, to a lesser extent, if the comment on the original issue about making more advanced tag control options in general is part of the plans for the future, i think it would set a bad precedent if this single control feature is tied to the FocusNextTab command, though it's not like the code couldn't be refractored to accomodate that if that becomes the case. either way, this is just my perspective as an outsider looking in (and i probably spent more words writing about this change than it would take to implement it lol), and i would be more than happy to change the code to reflect this change if it would help make the codebase clearer.

as for FocusPreviousEmptyTag, i feel like it wouldn't have that much of a use, as i feel like the main use case for FocusNextEmptyTag is when you just want a clean workspace and don't particularly care where it is. though, with how trivial it would be to implement, it might be a good idea regardless.